DB_Model.json

```diff
@@ -1,16 +1,631 @@
+                    "Rinat Neuroscience Corp",
+                    "Pfizer Inc."
+                ]
+            ],
+            "additionalItems": true,
+            "items": {
+                "$id": "#/properties/assignees/items",
+                "anyOf": [
+                    {
+                        "$id": "#/properties/assignees/items/anyOf/0",
+                        "type": "string",
+                        "title": "The first anyOf schema",
+                        "description": "An explanation about the purpose of this instance.",
+                        "default": "",
+                        "examples": [
+                            "Rinat Neuroscience Corp",
+                            "Pfizer Inc."
+                        ]
+                    }
+                ]
+            }
+        },
+        "examiners": {
+            "$id": "#/properties/examiners",
+            "type": "array",
+            "title": "The examiners schema",
+            "description": "An explanation about the purpose of this instance.",
+            "default": [],
+            "examples": [
+                [
+                    "Chandra, Gyan"
+                ]
+            ],
+            "additionalItems": true,
+            "items": {
+                "$id": "#/properties/examiners/items",
+                "anyOf": [
+                    {
+                        "$id": "#/properties/examiners/items/anyOf/0",
+                        "type": "string",
+                        "title": "The first anyOf schema",
+                        "description": "An explanation about the purpose of this instance.",
+                        "default": "",
+                        "examples": [
+                            "Chandra, Gyan"
+                        ]
+                    }
+                ]
+            }
+        },
+        "sequenceCount": {
+            "$id": "#/properties/sequenceCount",
+            "type": "integer",
+            "title": "The sequenceCount schema",
+            "description": "An explanation about the purpose of this instance.",
+            "default": 0,
+            "examples": [
+                187
+            ]
+        },
+        "proteinId": {
+            "$id": "#/properties/proteinId",
+            "type": "string",
+            "title": "The proteinId schema",
+            "description": "An explanation about the purpose of this instance.",
+            "default": "",
+            "examples": [
+                "PCSK9"
+            ]
+        },
+        "claimedResidues": {
+            "$id": "#/properties/claimedResidues",
+            "type": "array",
+            "title": "The claimedResidues schema",
+            "description": "An explanation about the purpose of this instance.",
+            "default": [],
+            "examples": [
+                [
+                    {
+                        "seqNoId": 53,
+                        "numbers": [
+                            1,
```

Comment:
should we include the aminoacid code here too? maybe each one would be an object with sequence position and aminoacid string

Reply:
Sure can. This would be an additional step in the `ta` process to lookup the amino acid letter code from the sequence document